### PR TITLE
STAR-151: search bar and filters css changes - child PR

### PR DESCRIPTION
**Pull request by chandan-30**
### Description
This PR is a sub PR for https://github.com/brainstormforce/starter-templates-components/pull/43 which consists of search bar and filters css changes

### Screenshots
https://d.pr/v/Ky4o64

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: src/search/style.scss [hunk 1]
```diff
@@ -22,11 +22,11 @@
 			border: var( --stc-search-input--focus-border, none );
 		}
 
-		&:hover,
-		&:active,
-		&:focus {
-			box-shadow: 0px 2px 6px -1px rgb(0 0 0 / 6%), 0px 4px 8px -1px rgb(0 0 0 / 10%);
-		}
+		// &:hover,
```
Comment (vrundakansara) [inline src/search/style.scss]: Do we need commented code?

**Review by vrundakansara — CHANGES_REQUESTED** [review]

File: src/search/style.scss (hunk 1 above)
Comment (chandan-30) [reply src/search/style.scss]: I was not sure whether to let it be or not as it was not specifically mentioned anywhere in figma regarding hover / active / focus of searchbar !!!

Comment (vrundakansara) [reply src/search/style.scss]: Please get the necessary inputrs from the designer and implement the changes

**Comment by vrundakansara** [comment]
Is this PR ready for review?

File: src/search/style.scss (hunk 1 above)
Comment (chandan-30) [reply src/search/style.scss]: added it back with new value.

**Review by vrundakansara — APPROVED** [review]

**Review by sadanandl — APPROVED** [review]

**Pull request merged**